app/assets/stylesheets/ama_layout/layout_components/banners.scss

```diff
@@ -23,6 +23,22 @@
     color: $white;
   }
 
+  &__container{
+    @include grid-row;
+    @include clearfix;
+
+    &--two{
+      @include large-2;
+      @include small-12;
+    }
+
+    &--ten{
+      @include large-10;
+      @include small-12;
+      margin-top: 30px;
+    }
+  }
+
```

Comment:
I'm not sure if maybe you just haven't pushed your changes but you will want to update these classes. See the comments I made on your membership PR (banner__image-container, banner__content-container)